components/CSS/component.css

```diff
@@ -7,34 +7,35 @@
     background-color: var(--color-primary);
     color: var(--white);
     font-family: var(--font-dosis);
+    position: sticky;
+    top: 0;
+    align-self: flex-start;
+    z-index: 9999;
 }   
 
 .content-box {
     display: flex;
+    width: 100%;
+    height: 100vh;
     flex-wrap: nowrap;
     
 }
 .side-container {
     width: 30%;
-    max-width: 300px;
-    padding: var(--size-two);
+    max-width: 200px;
```

Comment:
it will be good if you can use same units for every css element rather than using absolute unit like px